### PR TITLE
Late changes (from my previews PR)

### PR DESCRIPTION
**Pull request by leoheck**
@jcyrax my PR wasn't finished when you approved. I was having issues with the openEMS. Now I was able to fix it. Somehow, I put a different/wrong/old/outdated `libCSXCAD.so.0` library on my system, and then the whole openEMS tools were not working anymore.

Ok. Now it is fixed, tested, and also improved a little bit.
- I updated the README with the initial info about the config.json you've explained to me.
- I fixed the errors in the .json of the 2nd example
- I improved the execution outputs by removing the octave banner and the octave clear screen command 

Everything is working just fine.
Now I will try to use this with my boards... probably it won't be easy because pcbmodelgen was not completely build for Kicad 5.*

### CODE REVIEW

**Comment by leoheck** [comment]
It also tries to fix this https://github.com/jcyrax/pcbmodelgen/issues/5

File: source/main.cpp [hunk 1]
```diff
@@ -31,6 +31,8 @@
 
 #define s_VERSION "0.2.0"
 
+int g_ERROR { 0 }; // zero initialized
```
Comment (jcyrax) [inline source/main.cpp:34]: Don't need to comment language syntax. Its obvious what happens here.

Comment (jcyrax) [inline source/main.cpp:34]: Don't need to comment language syntax. Its obvious from expression that its zero initialized.

**Comment by leoheck** [comment]
No. I am done with this PR.

I just have a question. I would like to use this tool with bigger boards, did you tried something like that? Do you have any example that is more complex of the examples available in the repo to share?

**Comment by jcyrax** [comment]
> No. I am done with this PR.
> 
> I just have a question. I would like to use this tool with bigger boards, did you tried something like that? Do you have any example that is more complex of the examples available in the repo to share?

I've tried on complex boards, but only to test conversion functionality. I've done simulations for small boards like RF tags and car immobilizers. 

Once you have more traces and copper that is not related to actual point of interest you have a drastic increase in mesh lines and by that also simulation times. You'll notice that OpenEMS doesn't have support for mesh subgrid. Besides that you'll have to cope with problems related to using zero thickness copper layers. Normally you would use mesh lines at offset from actual copper boundary, but now you have mesh lines that are from geometry that's in different location on PCB. This basically means you have to simulate actual copper thickness so now you'll have extra mesh lines on Z axis.

The best approach is to make your problem very simple. Simulate only actual RF problems. 

As a final note I should say that its not trivial to get correct results for even known RF problems. So keep this in mind when using results from all this.

Also I'm no expert in FDTD so if you have more general questions regarding OpenEMS read trough https://openems.de because there are a lot of valuable information.

**Comment by leoheck** [comment]
Good to hear this. My main idea was to test everything that I can use to simulate Kicad PCBs for EMI and RF emissions.

Now I am thinking that pcbmodelgen could have a way to pass the trace name, then it would generate the board with that trace only to make the simulations easy.

**Pull request merged**